bundles/org.openhab.binding.sunsynk/README.md

```diff
@@ -0,0 +1,245 @@
+Number:Dimensionless        InverterControlWorkMode     "System Work Mode 0, 1 or 2"                   {channel="sunsynk:inverter:a1a6340bc0:E4701229R3312211229948:Inverter-control-work-mode", widget="oh-slider-card",listWidget="oh-slider-item"[title="Inverter Work Mode",subtitle="0 - Selling, 1 - Zero-Export or 2 - Limted to Home", min=0, max=2,step=1]}
+Number:Dimensionless        InverterControlPattern      "System Mode Energy Pattern 0 or 1"            {channel="sunsynk:inverter:a1a6340bc0:E4701229R3312211229948:Inverter-control-energy-pattern", widget="oh-slider-card",listWidget="oh-slider-item"[title="Inverter Energy Pattern",subtitle="0 - Battery or 1 - Load", min=0, max=1,step=1]}
+```
+
+## DateTime Widget
+
+The items file above adds Metadata: Default Standalone widget: [rlk_datetime_standalone](https://community.openhab.org/t/datetime-standalone-widget/127966) to the DateTime items. Only the time portion of the DateTime item is important.
+
+Be sure to understand the time zone set up for the inverter, this can either be synchronised with Sun Synk servers, which in the UK at least applies daylight saving, or free-wheeling locally. The times set in the DateTime items using the widget are not adjusted to any time zone and are sent to the SunSynk API as Strings where they will be applied directly to your inverter. This is in contrast to other solar / energy  APIs that use Zulu (GMT) time.
+## Installation
+
+Place the `.jar` file into your `addons` directory. See the "Through manually provided add-on" section of [Installation of Add-ons](https://www.openhab.org/docs/configuration/addons.html).
+
+## Trouble Shooting 
+This first version of the binding has been built against 4.2.0-SNAPSHOT - Build #3989, currently its only  available as a `.jar` file 
+```java
+org.openhab.binding.sunsynk-4.2.0-SNAPSHOT
+```
+Although the binding is written to find multiple Inverters for each Sun Synk Connect account, testing has been against an account with a single inverter.
+
+Sun Synk account and inverter versions are:
+- Gateway Model EESW-D205
+- Gateway COMM Type Wi-Fi
+- Gateway HW AEW2-0005-03
+- Gateway SW 4790123B24R
+- Inverter 8.0 kW
+- Inverter Type Energy Storage
+- Inverter SW M 5.3.8.4 / S 1.5.1.5 / C E.4.3.0
+- Connect App Version 1.11.12
+
+### Debugging
+
+After  installation, to gain further information on any issues you encounter  you  can turn on Debug [Logging](https://www.openhab.org/docs/administration/logging.html)  either  through the [karaf console](https://www.openhab.org/docs/administration/console.html) or through the openHAB UI.
+
+Via Console:
+```java
+log:set DEBUG org.openhab.binding.sunsynk
+```
+
+Via UI: 
+
+Log on to your local openHAB as admin and follow the menus.
+
+- Settings > Add-on-Store > Bindings
+- Scroll down to find the binding and click on its title, click the cog and select the level "Debug" on the right.
+
+You should now be able to see debug data in the `openhab.log`.
+
+
+
+
+
+
+
+
```

Comment:
Can be removed as the provided links will always have the best possible instruction and should be the single source.

Reply:
Done